index.html

```diff
@@ -11,7 +11,7 @@
 startdate: 2016-05-09      # use YYYY-MM-DD format like 2015-01-01
 enddate: 2016-05-10      # use YYYY-MM-DD format like 2015-01-02
 instructor: ["Luke Johnston", "Ricardo Harripaul", "Derek Howard", "Lina Tran"] # list of names like ["Kay McNulty", "Betty Jennings", "Betty Snyder"]
-helper: ["Madeleine Bonsma", "Justin Kenney", "Sophie McGibbon", "Dano Morrison"]     # list of names like ["Marlyn Wescoff", "Fran Bilas", "Ruth Lichterman"]
+helper: ["Madeleine Bonsma", "Justin Kenney", "Sophie McGibbon", "Dano Morrison", "John Ladan", "Lina Tran", "Colleen Gillon", "Thomas He", "Derek Howard", "Elliott Sales de Andrade", "Luke Johnston", "Maria Tibbo", "Zsu Buchwald"]     # list of names like ["Marlyn Wescoff", "Fran Bilas", "Ruth Lichterman"]
```

Comment:
Thanks, I meant to not include them but clearly did not succeed. I'll change that!